files/clean-salt

```diff
@@ -19,9 +19,15 @@ echo "Purging Salt config..."
 
 if [[ ! -d "$SDW_SALT_DIR" ]]; then
     sudo rm -rf ${SDW_SALT_DIR}
+
+    # Can be removed in future
     sudo rm -rf ${SALT_DIR}/launcher
+
+    # We no longer store salt files directly in /srv/salt, so these next
+    # 3 checks can be removed at 4.2 cutover
     sudo find ${SALT_DIR} -maxdepth 1 -type f -iname 'fpf*' -delete
```

Comment:
We can remove them now in that case - there won't be a supported in-place upgrade path.

Reply:
[yes - it was for developer convenience but I'm happy to remove it now]